pkg/branding/brandedlogo_test.go

```diff
@@ -1,75 +1,40 @@
 package branding
 
 import (
-	"embed"
-	"encoding/base64"
 	"testing"
 
-	"github.com/stackrox/rox/pkg/testutils/envisolator"
-	"github.com/stackrox/rox/pkg/utils"
-	"github.com/stretchr/testify/suite"
+	"github.com/cloudflare/cfssl/scan/crypto/sha256"
+	"github.com/stretchr/testify/assert"
 )
 
 var (
-	//go:embed files/red_hat_acs_logo_rgb.png
-	logoRHACStest       embed.FS
-	logoRHACSBase64test = func() string {
-		bytes, err := logoRHACStest.ReadFile("files/red_hat_acs_logo_rgb.png")
-		utils.Must(err)
-		return base64.StdEncoding.EncodeToString(bytes)
-	}()
-	//go:embed files/stackrox_logo_wide_dkblue.png
-	logoStackRoxtest       embed.FS
-	logoStackRoxBase64test = func() string {
-		bytes, err := logoStackRoxtest.ReadFile("files/stackrox_logo_wide_dkblue.png")
-		utils.Must(err)
-		return base64.StdEncoding.EncodeToString(bytes)
-	}()
+	logoRHACSBase64hash    = [32]uint8{0x7f, 0x3e, 0x82, 0x96, 0x3a, 0x70, 0x5c, 0x41, 0xca, 0xc0, 0x96, 0xb5, 0x16, 0xde, 0xe0, 0x68, 0xec, 0x1e, 0x86, 0x93, 0xd5, 0x5e, 0xc6, 0xd8, 0x36, 0x54, 0x6b, 0x12, 0xc6, 0x17, 0xe1, 0x95}
```

Comment:
I have two suggestions:
1. Please look for a way to convert byte array to hexademic string, i.e. here it would be `"7f3e82963a705c41cac096b516dee068ec1e8693d55ec6d836546b12c617e195"`. It is just more familiar to read has this way.
2. Add a code comment for future-us how to regenerate these hashes in case files change.
For that I came up with this, I guess you can find a simpler way
```sh
find pkg/branding/files -type f -print -exec bash -c 'base64 -w0 {} | sha256sum' \;
```

Reply:
Done